src/app/login/page.tsx

```diff
@@ -0,0 +1,123 @@
+'use client';
+
+import React, { useState } from 'react';
+import Image from 'next/image';
+import GoogleIcon from '@/../public/images/google-icon.png';
+import Link from 'next/link';
+
+const Login = () => {
+  const [toggleEye, setToggleEye] = useState(false);
+
+  const toggleEyeHandler = () => {
+    setToggleEye((prev) => !prev);
+  };
+
+  return (
+    <>
+      <div className="mt-16 h-[calc(100vh-17rem)] md:h-[calc(100vh-19rem)] bg-[#EEE7DF] md:bg-white">
```

Comment:
for the bg-[#EEE7DF], check the tailwind.config, it's already defined there so put bg-vool-orange or the corresponding name